mixer/pkg/adapter/instanceUtil.go

```diff
@@ -43,6 +43,8 @@ func Stringify(v interface{}) string {
 			return strconv.FormatFloat(vv, 'f', -1, 64)
 		case bool:
 			return strconv.FormatBool(vv)
+		case []byte:
```

Comment:
I'm a bit confused about needing this, as there is a `case net.IP` already. IIRC, we recently "fixed" how IPs flowed through Mixer. Did we miss something that leads to `[]byte` in `logging.Entry`s  (or did we settle on `[]byte` and my memory is foggy)?

@kyessenov any thoughts here?

Reply:
There should never be `net.IP` in the attribute bag. See https://github.com/istio/istio/blob/release-1.1/mixer/pkg/attribute/bag.go#L25. It should always be `[]byte`. The two types are aliased, but not the same in case switches like this. This is just another example of dynamic typing that was not updated due to confusion. It is safe to remove `net.IP` case here.

Reply:
Thanks.. will remove net.IP case from here